infra-as-code/bicep/modules/vnetPeering/parameters/vnetPeering.parameters.min.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "parDestinationVirtualNetworkID": {
```

Comment:
```suggestion
    "parDestinationVirtualNetworkId": {
```

Reply:
Please see latest update to PR and resolve this conversation.